pyproject.toml

```diff
@@ -11,9 +11,10 @@ python = "^3.8"
 fastapi = "^0.61.1"
 uvicorn = "^0.12.2"
 requests = "^2.24.0"
-python-dotenv = "^0.14.0"
+python-dotenv = "^0.15.0"
 pymongo = "^3.11.2"
 boto3 = "^1.16.61"
+dmpy = { git = "https://github.com/ideafast/dmpy.git", branch = "ft_upload" }
```

Comment:
Note: this branch should be changed to "master" (or removed) once [this issue](https://github.com/ideafast/dmpy/pull/2) is merged.

Reply:
Agreed, I would argue to tag a version to the branch at that stage to allow developments on the master branch without breaking it's implementation here. E.g.,
```python
dmpy = { git = "https://github.com/ideafast/dmpy.git", tag = "v1.0" }
```

Reply:
Can we merge in this change after merging the PR on the dmpy side? Then we have that in one go.